runtime/src/accounts.rs

```diff
@@ -440,6 +442,10 @@ impl Accounts {
                         }
                         owner_index
                     };
+                    if disable_builtin_loader_ownership_chains {
```

Comment:
If loader chains are disabled why do we loop 5 times here?

Reply:
We don't the return statement below breaks the loop in the first iteration. It as the easiest way to have a feature gate. If I had only set it to one iteration, it could still have run into `CallChainTooDeep` which should not be possible after activation anymore.